avaframe/com1DFA/com1DFA.py

```diff
@@ -958,6 +958,8 @@ def placeParticles(massCell, indx, indy, csz, massPerPart, rng, initPartDistType
         if rng.random(1) < proba:
             nPart = nPart + 1
         nPart = np.maximum(nPart, 1)
+        if (massCell / nPart) / massPerPart >= 1.5:
```

Comment:
Where does the 1.5 come from? Looks like a parameter -> ini file

Reply:
the particles that are too big will anyways be splitted immediately so that is why we do it here